Jenkinsfile

```diff
@@ -0,0 +1,85 @@
+pipeline {
+    agent {
+        kubernetes {
+            yaml """
+apiVersion: v1
+kind: Pod
+metadata:
+  labels:
+    name: saraswati-dashboard 
+spec:
+  containers:
+  - name: node
+    image: node:13.10.1-alpine3.11
+    command:
+    - cat
+    tty: true
+  - name: kaniko
+    image: gcr.io/kaniko-project/executor:debug
+    imagePullPolicy: Always
+    command:
+    - /busybox/cat
+    tty: true
+    volumeMounts:
+      - name: jenkins-docker-cfg
+        mountPath: /kaniko/.docker
+  volumes:
+  - name: jenkins-docker-cfg
+    projected:
+      sources:
+      - secret:
+          name: mh-docker-hub
+          items:
+            - key: config.json
+              path: config.json
+"""
+        }
+    }
+    environment {
```

Comment:
Let's touch base with Mike or Elijah to see where and how we get these environmental variables. I don't think we should be declaring environmental variables inside this file at all.